### PR TITLE
Issue 7 - Add new note warning

### LINKED ISSUE #7: Don’t display notes if there are no notes.
Update reminder component to have a conditional that will only display notes if notes exist, with some sort of message on the default page that indicates to the user they should create a new note.   

Tests or it didn't happen.

### PR DESCRIPTION
**Pull request by jksmall0631**
## Purpose

Update reminder component to have a conditional that will only display notes if notes exist, with some sort of message on the default page that indicates to the user they should create a new note.

Tests or it didn't happen.

[Issue #7](https://github.com/turingschool-projects/1610-remember-1/issues/7)

## Approach

We started by not displaying mirage. We wanted our app to only display notes that the users create. We then made an if statement conditional to show and hide a 'create a new note' message. Next we worked on writing some tests to show that our app does exactly what we want it to.
### Learning

Just took a look at the ember docs, per usual.

[Ember docs](https://guides.emberjs.com/v2.2.0/templates/conditionals/)

### Open Questions and Pre-Merge TODOs

- [x]  remove mirage
- [x]  add conditional to template
- [x]  add acceptance tests for our conditionals

### Test coverage 

We tested to see if our 'create a new reminder' message appeared when there are no reminders.

### Follow-up tasks

- [Issue #11 ](https://github.com/turingschool-projects/1610-remember-1/issues/11)

#### Before
screen shot 2017-02-11 at 7 23 04 pm

#### After
screen shot 2017-02-12 at 6 31 38 pm
screen shot 2017-02-12 at 6 32 45 pm
screen shot 2017-02-12 at 6 15 59 pm

close #7

### CODE REVIEW

File: app/templates/components/reminders-list.hbs [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+
+{{#link-to "reminders.new-reminder"}}
+  <button class='spec-reminder--new'>
+    new
+  </button>
+{{/link-to}}
 
 <ul>
+  {{#if model}}
   {{#each model as |reminder|}}
```
Comment (Tman22) [inline app/templates/components/reminders-list.hbs]: Everything in the if block should be indented one tab or 2 spaces.

File: tests/acceptance/reminder-list-test.js [hunk 2]
```diff
@@ -29,3 +29,38 @@ test('clicking on an individual item', function(assert) {
...
+  visit('/');
+  click('.spec-reminder--new');
+  fillIn('.spec-input-title', 'Bananas')
+  fillIn('.spec-input-date', '12/23/16')
+  fillIn('.spec-textarea-notes', 'This shit is bananas, B.A.N.A.N.A.S.')
+  click('.spec-reminder-add--submit');
+
+  andThen(function() {
+    assert.equal(currentURL(), '/reminders/new-reminder', 'current url is /reminders/new-reminder');
+    assert.equal(Ember.$('.spec-input-title').text().trim(), Ember.$('.spec-reminder-card--title:last').text().trim(), 'adds item to reminder list on submit')
```
Comment (Tman22) [inline tests/acceptance/reminder-list-test.js]: Here you should check that the card :last has text "Bananas" and or "12/23/16"

Comment (Tman22) [reply tests/acceptance/reminder-list-test.js]: Also be consistent with semicolons.

**Comment by jksmall0631** [comment]
@Tman22 @martensonbj Made the changes requested.

**Review by martensonbj — APPROVED** [review]

**Pull request merged**